src/PrestaShopBundle/Command/ListCommandsAndQueriesCommand.php

```diff
@@ -62,27 +67,130 @@ public function configure()
 
-            $output->writeln(++$key . '.');
-            $output->writeln('<blue>Class: </blue><info>' . $commandDefinition->getClassName() . '</info>');
-            $output->writeln('<blue>Type: </blue><info>' . $commandDefinition->getCommandType() . '</info>');
-            $output->writeln('<comment>' . $commandDefinition->getDescription() . '</comment>');
-            $output->writeln('');
+            if ($this->isFormatSimple) {
+                $output->writeln('<info>' . $commandDefinition->getClassName() . (!empty($cqrsEndpointURI) ? ' OK' : ' NOT OK') . '</info>');
+            } else {
+                $output->writeln(++$key . '.');
+                $output->writeln('<blue>Class: </blue><info>' . $commandDefinition->getClassName() . '</info>');
+                $output->writeln('<blue>Type: </blue><info>' . $commandDefinition->getCommandType() . '</info>');
+                $output->writeln('<blue>API: </blue><info>' . $cqrsEndpointURI . '</info>');
+                $output->writeln('<comment>' . $commandDefinition->getDescription() . '</comment>');
+                $output->writeln('');
+            }
         }
 
         return 0;
     }
+
+    private function handleOptions(InputInterface $input): void
+    {
+        if ($input->getOption('domain') !== []) {
+            $this->filterCQRS($input->getOption('domain'));
+        }
+
+        if ($input->getOption('format') === 'simple') {
+            $this->isFormatSimple = true;
+        }
+    }
+
+    /**
+     * @param string[] $filters
+     */
+    private function filterCQRS(array $filters): void
+    {
+        $this->commandAndQueries = array_filter($this->commandAndQueries, function (string $currentCQRS) use ($filters) {
+            foreach ($filters as $filter) {
+                // We append a backslash behind the filter to find only exact matches
+                if (str_contains($currentCQRS, $filter . '\\') !== false) {
+                    return true;
+                }
+            }
+
+            return false;
+        });
+    }
+
+    /**
+     * This method rebuild the resources from the api platform attributes.
+     *
+     * @return ResourceMetadataCollection[]
+     */
+    private function getResourceList(): array
+    {
+        $resourceMetadataCollection = [];
+        foreach ($this->resourceNameCollectionFactory->create() as $resourceClass) {
+            $resourceMetadataCollection[] = $this->resourceMetadataFactory->create($resourceClass);
+        }
+
+        return $resourceMetadataCollection;
+    }
+
+    /**
+     * This method takes the filtered list of routes, and checks it against the list of CQRS.
+     * The CQRS that have a route with the correct method are implemented, so we return the URI of the endpoint.
+     */
+    private function getCQRSEndpointURI(CommandDefinition $commandDefinition): string
+    {
+        $domain = explode('\\', $commandDefinition->getClassName())[4];
```

Comment:
This can fail if `getClassName()` returns a bad value. You could get a illegal offset error. You need to make sure `explode()` result contains at least 4 items (you can throw an exception if this is not the case)